### DISCUSSION TITLE
AIR SDK licensing

### DISCUSSION THREAD

**Discussion by vitalysoftech** [airsdk/Adobe-Runtime-Support#1435] [General]
Hello,

sorry for off-topic, but we cannot find the other way how to reach someone to answer our questions regarding the AIR SDK licensing. We wrote to official AIR SDK support e-mail, wrote to another e-mail of Andrew Frost: Andrew.Frost@harman.com, but still didn't get any answer. We also called to official HARMAN phone number, but this is didn't help as well. The best way to address our licensing questions would be to reply on the e-mail sent from ekostin@softech.ru (Eugene Kostin) account. If the e-mail message cannot be found, please add it to white list and send some e-mail, so we could answer and proceed discussion there.

Thanks!

Regards,
Vitaly Shlyaga, 
Software Technologies LLC

**Comment by ajwfrost** [comment]
Hi

Not sure what's going on, I can't find any emails from softech in my inbox or the support email account..
Will email Eugene via the above and see whether it works via a response!

thanks
  Andrew

**Reply by vitalysoftech** [reply]
Hi Andrey,

we got your e-mail and will reply today. Please let us know here if you don't receive e-mail again.

Thanks!

Regards,
Vitaly

**Comment by vitalysoftech** [comment]
Hi Andrew,

can I check here on one licensing case? If a user makes builds on two different computers does he needs one or two licenses? For example, there can be 2 build servers, where user must log in to manually run a build script from console. Or it can be a user who develops for Win and Mac and thus needs to execute builds on Win and Mac machines.

Regards,
Vitaly
